.devcontainer/devcontainer.json

```diff
@@ -1,22 +1,32 @@
 {
-    "extensions": [
-		"golang.go"
-	],
-	"build": {
-		"dockerfile": "Dockerfile",
-		"args": {
-			"VARIANT": "1.18"
-		}
+	"image": "mcr.microsoft.com/devcontainers/go:1.18",
+	"features": {
+		"ghcr.io/devcontainers/features/github-cli:1": {
+			"version": "latest"
+		},
+		"ghcr.io/devcontainers/features/sshd:1": {
+            "version": "latest"
+        }
 	},
-	"settings": {
-		"go.toolsManagement.checkForUpdates": "local",
-		"go.useLanguageServer": true,
-		"go.gopath": "/go"
+	"remoteUser": "vscode",
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"golang.go"
+			],
+			"settings": {
+				"go.toolsManagement.checkForUpdates": "local",
+				"go.useLanguageServer": true,
+				"go.gopath": "/go",
+				"search.exclude": {
+					"vendor/**": true
```

Comment:
We don't vendor anything in this repo, so I don't think this exclude pattern is needed.

Reply:
Cool, let's get rid of it then